data/json/items/melee.json

```diff
@@ -406,7 +406,7 @@
   {
     "type": "GENERIC",
     "id": "nailboard",
-    "name": "nail board",
+    "name": "nailboard",
```

Comment:
I think this line should get reverted.

Reply:
You'd rather have "nail board trap" instead of "nailboard trap" too, or do you just not mind the inconsistancy?